docs/modules/clients/pages/java.adoc

```diff
@@ -59,19 +143,103 @@ mapCustomers.put("2", new Customer("Ali", "Selam"));
+access them through a proxy on the client side.
+
+=== Use Map
+
+You can use any distributed map object with the client, as follows:
+
+[source,java]
+----
+Imap<Integer, String> map = client.getMap("myMap");
+
+map.put(1, "John");
+String value= map.get(1);
+map.remove(1);
+----
+
+The `addLocalEntryListener()` and `localKeySet()` methods are not supported because locality is ambiguous for the client. For more information, see xref:data-structures:map.adoc[].
+
+=== Use MultiMap
+
+You can use a distributed multiMap object with the {java-client}, as follows:
+
+[source,java]
+----
+MultiMap<Integer, String> multiMap = client.getMultiMap("myMultiMap");
+
+multiMap.put(1,"John");
+multiMap.put(1,"Mary");
+
+Collection<String> values = multiMap.get(1);
+----
+
+The `addLocalEntryListener()`, `localKeySet()` and `getLocalMultiMapStats()` methods are not
+supported because locality is ambiguous for the client. For more information, see xref:data-structures:multimap.adoc[].
+
+=== Use Queue
+
+You can use a distributed Queue object with the {java-client}, as follows: 
+
+[source,java]
+----
+IQueue<String> myQueue = client.getQueue("theQueue");
+myQueue.offer("John")
+----
+
+The `getLocalQueueStats()` method is not supported because locality is ambiguous for the client.
+For more information, see xref:data-structures:queue.adoc[]. 
+
+=== Use Topic
+
+The `getLocalTopicStats()` method is not supported because locality is ambiguous for the client.
+
+=== Other supported distributed structures
+
+The distributed data structures listed below are also supported by the {java-client}.
+The logic is the same for both member and client side, so see the specific sections for more information on usage.
 
-=== Client Cluster Routing Modes
+* xref:data-structures:replicated-map.adoc[Replicated Map]
+* xref:data-structures:list.adoc[List]
+* xref:data-structures:set.adoc[Set]
+* xref:data-structures:iatomiclong.adoc[IAtomicLong]
+* xref:data-structures:iatomicreference.adoc[IAtomicReference]
+* xref:data-structures:icountdownlatch.adoc[ICountDownLatch]
+* xref:data-structures:isemaphore.adoc[ISemaphore]
+* xref:data-structures:flake-id-generator.adoc[FlakeIdGenerator]
+* xref:data-structures:fencedlock.adoc[Lock]
+* xref:data-structures:cpmap.adoc[CPMap]
 
+== Configure the client
+=== Client cluster routing modes
+// decide placement
 The cluster routing mode specifies how the client connects to the cluster. It can currently be used only with Java and .NET clients.
+// check standalone?
+
+// NOTE: This section is only applicable to the {java-client}.
+// NOTE: The {java-client-new} only supports smart routing in `Unisocket mode`. You need to disable Smart routing for clients which
+// want to use temporary permissions defined in a member. 
+// For more info, see xref:security:native-client-security.adoc#handling-permissions-when-a-new-member-joins[Handling permissions].
```

Comment:
as far as the Java Standalone client has routing modes, maybe it's because this documentation is for 5.5.0-beta version?

Reply:
Yeah, I saw the conversation here https://github.com/hazelcast/hz-docs/pull/1240#discussion_r1741811784 so put this in ready to be uncommented once confirmed:

@alexey-agapov-hazelcast please confirm

Reply:
@oliverhowell the Java Standalone client does support all three routing modes in the current version

Reply:
I'll uncomment the section but see Serkan's question about unisocket mode.

Reply:
This text is out now anyway so resolving